src/pybind/tensor_buffer.cpp

```diff
@@ -1,39 +1,31 @@
 #include "tensor_buffer.hpp"
+#include <iostream>
```

Comment:
I dont see where iostream is used, I might be wrong though

Reply:
Oversight on my part --- reveals my super secret `std::cout` debugging method ;)